src/runtime/StaticStrings.cpp

```diff
@@ -116,6 +116,7 @@ void StaticStrings::initStaticStrings()
     INIT_STATIC_STRING(intlDotCollator, "Intl.Collator");
     INIT_STATIC_STRING(intlDotLocale, "Intl.Locale");
     INIT_STATIC_STRING(intlDotRelativeTimeFormat, "Intl.RelativeTimeFormat");
+    INIT_STATIC_STRING(intlDotDisplayNames, "Intl.DisplayNames");
```

Comment:
Would you please keep the alphabetical order?

Reply:
I've updated my pr